lib/plugins/chest.js

```diff
@@ -12,7 +12,9 @@ function inject (bot, { version }) {
       assert.ok(chestToOpen.type === mcData.blocksByName.chest.id ||
                 chestToOpen.type === mcData.blocksByName.ender_chest.id ||
                 chestToOpen.type === mcData.blocksByName.trapped_chest.id ||
-                (mcData.blocksByName.barrel && chestToOpen.type === mcData.blocksByName.barrel.id))
+                (mcData.blocksByName.barrel && chestToOpen.type === mcData.blocksByName.barrel.id) ||
+                (mcData.blocks[chestToOpen.type].name.endsWith('_shulker_box')))
```

Comment:
there's no point in having the parentheses here.

Reply:
You are correct. I have removed them